examples/Waveshare_7_5/Waveshare_7_5.ino

```diff
@@ -498,58 +499,38 @@ boolean SetupTime() {
   return TimeStatus;
 }
 //#########################################################################################
+// required an update of all lang_xx.h files 
 boolean UpdateLocalTime() {
   struct tm timeinfo;
-  char   output[30], day_output[30];
-  char   day_number[30], day_year[30];
-  char   day_lang[4], month_lang[4];
-  char   update_time[30];
-  while (!getLocalTime(&timeinfo, 5000)) { // Wait for 5-sec for time to synchronise
+  char   output[23], day_output[30];
+  char   update_time[9];            // change from 30 to 9      
+  while (!getLocalTime(&timeinfo, 5000)) {    // Wait for 5-sec for time to synchronise

```

Comment:
OK, but we could use just one char array. output[] but I want to retain capacity for 'Sat Apr-20-2019 12:31:45PM' which is what some US users prefer, so the minimum would be output[28], which is where the [3] came from as I added 2 for good measure in-case some adds for other formatting codes. My preference would be:
char   output[30];
And to change 'day_output' to 'output' to re-use the same array.

Reply:
But, having thought about this some more, I need to be convinced that the tm structure is working across all Arduino IDE versions, for example it did not work (issues) with 1.0.0, seems OK with 1.0.2 and 1.0.3rc-1, but not everyone ahs the latest IDE. Needs testing to confirm.

Reply:
Is it correct, you will use only one array? 
In the current code 
output will only be used für "updated: hh:mm:ss am" or pm ?